mobile/src/store/middlewares/notify-logger/notify-logger.ts

```diff
@@ -0,0 +1,18 @@
+import type { MiddlewareAPI, Middleware } from '@reduxjs/toolkit';
+import { NotificationType } from '~/common/enums/enums';
+import { notify } from '~/store/auth/actions';
+
+const notifyLogger: Middleware =
```

Comment:
Let's rename it to handleError, move it to store/middleware/handle-error/handle-error.middleware.ts and re-export from middleware/middleware.ts